apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

```diff
@@ -101,23 +111,24 @@ export default function Content({
       },
     });
   }, [account.id, navigation, parentAccount]);
-  const onPressInfo = useCallback(() => {
+
+  const onPressInfo = () => {
     setIsModalOpened(true);
-  }, []);
-  const onModalClose = useCallback(() => {
+  };
+
+  const onModalClose = () => {
     setIsModalOpened(false);
-  }, []);
-  const mainAccount = getMainAccount(account, parentAccount);
+  };
```

Comment:
As mentionned before, please keep the `useCallback`s because here you're creating a new reference at each render, meaning that each component having them as prop will rerender as well, even if nothing changed.

Reply:
useCallback don't prevent child component to rerender, it just doesn't recompute the callback inside inside the current component if none of the props have changed. Here the cost of calling useCallback is greater than recalculating the functions so it's not a los in perf